.travis.yml

```diff
@@ -15,14 +15,17 @@ env:
   matrix:
   - TEST_SUITE=unit
   - TEST_SUITE=plainbox
+  - TEST_SUITE="plainbox examples"
 
 install:
   # PPA for plainbox.
   - sudo apt-add-repository -y ppa:hardware-certification/public
   # PPA for snappy.
   - sudo apt-add-repository -y ppa:snappy-dev/tools-proposed
+  # PPA for qtmir.
+  - sudo apt-add-repository -y ppa:ubuntu-sdk-team/testing
   - sudo apt-get update -qq
-  - sudo apt-get install -qq build-essential dpkg-dev plainbox pyflakes python3-apt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-requests python3-testscenarios python3-yaml ubuntu-snappy-cli
+  - sudo apt-get install build-essential dpkg-dev plainbox pyflakes python3-apt python3-coverage python3-fixtures python3-jsonschema python3-mccabe python3-pep8 python3-requests python3-testscenarios python3-yaml qtdeclarative5-qtmir-plugin ubuntu-snappy-cli
```

Comment:
I was trying to satisfy the requirements. Didn't go anywhere, so I'll rollback this changes and just skip the qml example in vivid.